src/util-reducers/data-reducer.js

```diff
@@ -20,30 +20,40 @@ export function DataReducer(state_data, action) {
             }
 
 
-
         case "DELETE_PLAYLIST":
             return { ...state_data, playlists: [...state_data.playlists.filter((item) => item._playlistName !== action.payload.playlistName)] }
 
 
         case "ADD_TO_PLAYLIST":
-
             {
-                alert(`added to playlist ${action.payload.playlistName}`)
-                if (videoIsPresent(action.payload.video, action.payload.playlistName))
-                    return { ...state_data }
-
-                return {
-                    ...state_data,
-                    playlists: state_data.playlists.map((playlist) =>
-                        playlist._playlistName === action.payload.playlistName
-                            ? { ...playlist, playlistVideos: playlist.playlistVideos.concat(action.payload.video) } : playlist
-                    )
+                if (videoIsPresent(action.payload.video, action.payload.playlistName)) {
```

Comment:
suggestion: naming can be improved to **isVideoPresent**

Reply:
okay i will